config/init.lua

```diff
@@ -104,6 +105,10 @@ for _, key in pairs(keys) do
     config.register_key(key)
 end
 
+function cleanup_background()
+  os.execute("pkill way-cooler-bg")
+end
+
 
 -- Execute some code after Way Cooler is finished initializing
 function way_cooler_init()
```

Comment:
Lua starts up [after the compositor has initialized](https://github.com/Immington-Industries/way-cooler/blob/master/src/callbacks.rs#L329). Than init function [then calls](https://github.com/Immington-Industries/way-cooler/blob/master/src/lua/thread.rs#L126) this function (`way_cooler_init`). There's no need to send a `COMPOSITOR_READY` message, as the compositor is already ready.